pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -137,11 +137,19 @@ func (t *FSTree) iterate(depth uint64, curPath []string, prm common.IteratePrm)
 
 		if prm.LazyHandler != nil {
 			err = prm.LazyHandler(*addr, func() ([]byte, error) {
-				return os.ReadFile(filepath.Join(curPath...))
+				data, err := os.ReadFile(filepath.Join(curPath...))
+				if err != nil && os.IsNotExist(err) {
```

Comment:
https://pkg.go.dev/os#IsNotExist

> This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrNotExist). 

Not that it's critical for our usage pattern, but this patter might be reused by other code, so we better use proper APIs.

Reply:
sure, fixed